lib/resty/etcd/v3.lua

```diff
@@ -27,6 +27,8 @@ local decode_base64 = ngx.decode_base64
 local semaphore     = require("ngx.semaphore")
 local health_check  = require("resty.etcd.health_check")
 local pl_path       = require("pl.path")
+local require = require
+local pcall   = pcall
```

Comment:
```suggestion
local require       = require
local pcall         = pcall
```

Reply:
move `local require       = require` to the top of file

Reply:
@tzssangglass fixed.